main/common/settings.js

```diff
@@ -13,7 +13,7 @@ const store = new Store({
     highlightClicks: false,
     hideDesktopIcons: false,
     record60fps: false,
-    loopKaptures: true,
+    loopRecordings: true,
```

Comment:
Isn't it weird calling this `recordings`? The recordings are not looped, it's the exported Kaptures that are.